test/system/150-login.bats

```diff
@@ -241,6 +241,36 @@ function _test_skopeo_credential_sharing() {
     rm -f $authfile
 }
 
+@test "podman login -secret test" {
+      secret=$(random_string 10)
+      echo -n ${PODMAN_LOGIN_PASS} > $PODMAN_TMPDIR/secret.file
+      run_podman secret create $secret $PODMAN_TMPDIR/secret.file
+
+      run_podman login --tls-verify=false \
+               --username ${PODMAN_LOGIN_USER} \
+               --secret ${output} \
```

Comment:
This squicks me out a little: first, it's not immediately obvious what `$output` is, second, it would be so easy to accidentally insert a `run_podman load` or `run` or something. Suggestion: add `secretid=$output` immediately after the `secret create`. Otherwise LGTM. with @TomSweeneyRedHat's suggestions.